Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.10.2-stretch
+
+RUN apk add --update go
```

Comment:
Here can minimize the number of layers, there are other place can be optimized, can refer to https://docs.docker.com/develop/develop-images/dockerfile_best-practices

Reply:
Development environment using the image is different production environment, proper use of the base image of the software is easier to configure, follow-up maintenance cost can reduce a lot, just a little sacrifice file size.

If you want to encapsulate a large and complete image, a fat docker image, maybe `debian/ubuntu/centos` is a wise choice.

Reply:
@soulteary CentOS 7 is not an option right now, because deno (or some of third-party) now relies on `GLIBC_2.18`, but CentOS 7 only support `GLIBC_2.17`. It should be fixed in Centos 8.